Solutions/MISP2Sentinel/ReleaseNotes.md

```diff
@@ -0,0 +1,3 @@
+| **Version** | **Date Modified (DD-MM-YYYY)** | **Change History**                          |
+|-------------|--------------------------------|---------------------------------------------|
+| 0.0.1       | 29-07-2023                     | **Data Connector** Initial version of MISP2Sentinel with support for Upload Indicators API
```

Comment:
Please check the version. As per me it should be 3.0.0.

Reply:
Changed in https://github.com/Azure/Azure-Sentinel/pull/8658/commits/01daea3dfc141fed17c97beae3d61d04e16749cb